en/views/helpers/form.rst

```diff
@@ -1411,51 +1989,140 @@ Output:
+field errors and when necessary display customized error messages.
+
+Displaying Errors
+-----------------
+
 .. php:method:: error(string $fieldName, mixed $text, array $options)
 
-Shows a validation error message, specified by $text, for the given
-field, in the event that a validation error has occurred.
+* ``$fieldName`` - A field name in the form ``'Modelname.fieldname'``.
+* ``$text`` - Optional. A string or array providing the error message(s). If an
+  array, then it should be a hash of key names => messages.  Defaults to
+  ``null``.
+* ``$options`` - An optional array that can only contain a boolean with the key
+  ``'escape'``, which will define whether to HTML escape the
+  contents of the error message. Defaults to ``true``.
 
-Options:
+Shows a validation error message, specified by ``$text``, for the given
+field, in the event that a validation error has occurred. If ``$text`` is not
+provided then the default validation error message for that field will be used.
 
--  'escape' bool Whether or not to HTML escape the contents of the
-   error.
+Uses the following template widgets::
 
-.. TODO:: Add examples.
+    'error' => '<div class="error-message">{{content}}</div>'
+    'errorList' => '<ul>{{content}}</ul>'
+    'errorItem' => '<li>{{text}}</li>'
 
-.. php:method:: isFieldError(string $fieldName)
+The ``'errorList'`` and ``'errorItem'`` templates are used to format mutiple
+error messages per field.
 
-Returns ``true`` if the supplied $fieldName has an active validation
-error. ::
+Example::
 
-    if ($this->Form->isFieldError('gender')) {
-        echo $this->Form->error('gender');
+    // If in TicketsTable you have a 'notEmpty' validation rule:
+    public function validationDefault(Validator $validator)
+    {
+        $validator
+            ->requirePresence('ticket', 'create')
+            ->notEmpty('ticket');
+    }
+
+    // And inside Templates/Tickets/add.ctp you have:
+    echo $this->Form->text('ticket');
+
+    if ($this->Form->isFieldError('ticket')) {
+        echo $this->Form->error('ticket', 'Completely custom error message!');
     }
 
+If you would click the *Submit* button of your form without providing a value
+for the *Ticket* field, your form would output:
+
+.. code-block:: html
+
+    <input name="ticket" class="form-error" required="required" value="" type="text">
+    <div class="error-message">Completely custom error message!</div>
+
 .. note::
 
     When using :php:meth:`~Cake\\View\\Helper\\FormHelper::input()`, errors are
-    rendered by default.
+    rendered by default, so you don't need to use ``isFieldError()`` or call
+    ``error()`` manually.
+
+.. tip::
+
+    If you use a certain model field to generate multiple form fields via
+    ``input()``, and you want the same validation error message displayed for
+    each one, you will probably be better off defining a custom error message
+    inside the respective :ref:`validator rules<create-validators>`.
```

Comment:
@antograssiot , thanks for the feedback.
Yes, I've just realized that it's mistyped; the real label is "creating-validators".
I will fix it.

So, for the point above (about the quotes) what should I do in the end? I'm a bit fuzzy now, after your edit of your initial comment. :)

Indeed, one of the reasons why I went ahead and added them where they were missing, was the fact that they were already used in the document, on similar words (attribute names, element names, key names etc.) but not in a consistent fashion. So, I was trying to at least make the whole page look the same, not have the same word some times in quotes and some others not.
One other reason is that in most browsers, the simple back ticks yield a pretty mild result (they only change the font) but, visually, the results is not very easy to pick up so they might be missed. On top of that, if you have a word that is surrounded by quotes but not backticks, in the final result your straight quotes will be converted to curly quotes which kind of "alters" a bit their meaning; we tend to assume straight quotes exemplify a value as it should appear in code (e.g. a key name) whereas curly quotes might just mean you want to emphasize or cite something. But it might be just me.

Anyways, in short, if you guys think they should really go away, I can parse the whole page and eliminate them (save for code samples, of course), or else let me know.